src/resources/ResourceSnapshots/ResourceSnapshots.ts

```diff
@@ -44,6 +45,17 @@ export default class ResourceSnapshots extends Resource {
         return this.api.get<ResourceSnapshotType[]>(`${ResourceSnapshots.baseUrl}/access/resource`);
     }
 
+    /**
+     * Verifies that the authenticated user has the correct access level to the content of a specific snapshot and returns a list of missing privileges.
+     * @param snapshotId The identifier of the target snapshot.
+     * @param options
+     */
+    listMissingPrivileges(snapshotId: string, options: ValidateAccessOptions) {
+        return this.api.get<PrivilegeModel[]>(
+            this.buildPath(`${ResourceSnapshots.baseUrl}/${snapshotId}/access/resources`, options),
```

Comment:
Is it expected that this endpoint isn't in the public swagger? https://platform.cloud.coveo.com/docs?urls.primaryName=Migration#/